apps/css-workshop/pages/date-picker.html

```diff
@@ -113,60 +113,12 @@ <h2>Date</h2>
             aria-label="month-2020-08"
           >
             <div class="iui-calendar-week">
-              <div
-                class="iui-calendar-day-outside-month"
-                tabindex="0"
-                aria-label="day-26"
-                role="option"
-                aria-disabled="false"
-              >
-                26
-              </div>
-              <div
-                class="iui-calendar-day-outside-month"
-                tabindex="0"
-                aria-label="day-27"
-                role="option"
-                aria-disabled="false"
-              >
-                27
-              </div>
-              <div
-                class="iui-calendar-day-outside-month"
-                tabindex="0"
-                aria-label="day-28"
-                role="option"
-                aria-disabled="false"
-              >
-                28
-              </div>
-              <div
-                class="iui-calendar-day-outside-month"
-                tabindex="0"
-                aria-label="day-29"
-                role="option"
-                aria-disabled="false"
-              >
-                29
-              </div>
-              <div
-                class="iui-calendar-day-outside-month"
-                tabindex="0"
-                aria-label="day-30"
-                role="option"
-                aria-disabled="false"
-              >
-                30
-              </div>
-              <div
-                class="iui-calendar-day-outside-month"
-                tabindex="0"
-                aria-label="day-31"
-                role="option"
-                aria-disabled="false"
-              >
-                31
-              </div>
+              <div class="iui-calendar-day-outside-month"></div>
```

Comment:
Not necessary. These html pages are only meant to demonstrate styles.

Technically it would also not be necessary in React, because there is no content inside the div and no other a11y attributes. However, I've heard of cases where some screen-readers will announce "clickable" when event delegation is used, so `aria-hidden` prevents that.